SimulationOutputReport/measure.rb

```diff
@@ -531,29 +531,23 @@ def get_outputs(timeseries_frequency,
         next unless end_use.annual_output_by_system[sys_id] > 0
 
         ec_vars = ep_output_names.select { |name| name.include? Constants.ObjectNameWaterHeaterAdjustment(nil) }
-        dsh_vars = ep_output_names.select { |name| name.include? Constants.ObjectNameDesuperheaterEnergy(nil) }
 
         ec_adj = get_report_variable_data_annual_mbtu(['EMS'], ec_vars)
-        dsh_adj = get_report_variable_data_annual_mbtu(['EMS'], dsh_vars)
-        break if ec_adj + dsh_adj == 0 # No adjustment
+        break if ec_adj == 0 # No adjustment
 
-        end_use.annual_output_by_system[sys_id] += ec_adj + dsh_adj
+        end_use.annual_output_by_system[sys_id] += ec_adj
         if include_timeseries_end_use_consumptions
           ec_adj_timeseries = get_report_variable_data_timeseries(['EMS'], ec_vars, UnitConversions.convert(1.0, 'J', end_use.timeseries_units), 0, timeseries_frequency)
-          dsh_adj_timeseries = get_report_variable_data_timeseries(['EMS'], dsh_vars, UnitConversions.convert(1.0, 'J', end_use.timeseries_units), 0, timeseries_frequency)
           end_use.timeseries_output_by_system[sys_id] = end_use.timeseries_output_by_system[sys_id].zip(ec_adj_timeseries).map { |x, y| x + y }
-          end_use.timeseries_output_by_system[sys_id] = end_use.timeseries_output_by_system[sys_id].zip(dsh_adj_timeseries).map { |x, y| x + y }
         end
         break # only apply once
       end
 
-      # Can only be one desuperheater or solar thermal system
-      if desuperheater_vars.empty?
-        desuperheater_vars = ep_output_names.select { |name| name.include? Constants.ObjectNameDesuperheaterLoad(nil) }
-      end
+      # Can only be one solar thermal system
```

Comment:
Have we done a test w/ multiple HVAC systems attached to multiple desuperheaters? We should confirm it works correctly.

Reply:
I've done a manual test with two dshs + two cooling systems.